### DISCUSSION TITLE
Sort icon looking different than in demo

### DISCUSSION THREAD

**Discussion by pboese** [Power-Components/livewire-powergrid#1319] [Q&A]
Heya, thanks for the great package! It really makes my life easier. There's one tiny issue I can't really explain though.

On https://demo.livewire-powergrid.com/ the sort icon for an unsorted column looks great but in our environment it looks like this:

<img width="803" alt="Screenshot 2024-01-07 at 17 47 04" src="https://github.com/Power-Components/livewire-powergrid/assets/13555377/124f27aa-dfac-41a2-9985-a25de6cc6347">

Is there a trick to have it look like in the demo?

<img width="232" alt="Screenshot 2024-01-07 at 17 47 36" src="https://github.com/Power-Components/livewire-powergrid/assets/13555377/8b02a1e1-667a-4910-8a5f-06e40f7c3fb1">

The characters used are the exact same but for some mysterious reason they are displayed differently here...

Thanks a lot in advance!

**Comment by marineusde** [comment]
Do yo use bootstrap or tailwind?

**Reply by pboese** [reply]
I'm using Tailwind, thanks!

**Reply by marineusde** [reply]
Please select the element in your browser dev tools, in my case there is a 
```
<span>↓</span>
```

That is ``&#8595;`` in HTML, its set in the trait WithSorting in the method sortLabel. Did you override this method?

Other idea: type ``&#8595;`` in your app.blade.php and see how it look, maybe your browser or css code do other things.

**Reply by pboese** [reply]
Thanks a lot for your help, Henning!

`&#8593; &#8597; &#8595;` indeed results in the following output:

![Untitled 2](https://github.com/Power-Components/livewire-powergrid/assets/13555377/46677300-91ca-4eb0-92a3-2d713c9755b5)

It happens only in Chrome (Mac, latest), now I'm even more confused ...
